### PR TITLE
doing homework7

### PR DESCRIPTION
**Pull request by Hromy-Ann**

### CODE REVIEW

File: src/main/java/org/example/homework7/Task2.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+public class Task2 {
+    public static void main(String[] args) {
+        String email = "hromovaannaa@gmail.com";
+        boolean result = validateEmail(email);
+        System.out.println(email + " is" + (result ? "" : " not") + " a valid email");
+    }
+
+    private static boolean validateEmail(String email) {
+        Pattern p = Pattern.compile("^((?!\\.)[\\w-.]*[^.])(@\\w+)(\\.\\w+(\\.\\w+)?\\w)$");
```
Comment (alexshamrai) [inline src/main/java/org/example/homework7/Task2.java]: `Pattern pattern` looks more fancy)

File: src/main/java/org/example/homework7/Task2.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        Matcher m = p.matcher(email);
```
Comment (alexshamrai) [inline src/main/java/org/example/homework7/Task2.java]:
```suggestion
        Matcher matcher = p.matcher(email);
```

File: src/main/java/org/example/homework7/Task1.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        System.out.println("Initial string: " + initialString);
+        System.out.println("Characters to remove: " + Arrays.toString(charsToRemove));
+        System.out.println("Modified string: " + modifiedString);
+    }
+
+    private static String removeCharacters(String input, char[] charsToRemove) {
+        StringBuilder stringBuilder = new StringBuilder();
+        for (char c : input.toCharArray()) {
+            boolean skip = false;
+            for (char d : charsToRemove) {
```
Comment (alexshamrai) [inline src/main/java/org/example/homework7/Task1.java]: what is `d`? As for me using clear name (e.g. `charToRemove`) would provide more information

**Review by alexshamrai — APPROVED** [review]

File: src/main/java/org/example/homework7/Task1.java (hunk 3 above)
Comment (Hromy-Ann) [reply src/main/java/org/example/homework7/Task1.java]: Thank you, all renames are done.

**Pull request merged**